IDBOOKAPI/IDBOOKAPI/utils.py

```
@@ -313,40 +326,49 @@ def find_state(district_name):
     try:
-        regex_mb = RegexValidator(regex=r'^\+?1?\d{9,15}$')
+        regex_mb = RegexValidator(regex=r"^\+?1?\d{9,15}$")
         regex_mb(mobile_number)
         return True
     except Exception as e:
         return False
 
+
 def shorten_url(original_url):
     try:
         res = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)